classes/3.html

```diff
@@ -86,22 +89,10 @@ <h3>¿Que es asincronía?</h3>
             </code></pre>
             <iframe data-src="../ejemplos/async/blocking.html" height="100px" width="800px"> </iframe>
             <aside class="notes">
-              <p>
-                El código que vemos acá corre sobre un html con un botón. En la primer linea vemos
-                que lo selecciona y en la segunda se agrega un manejador para el evento click. La
-                función que se define entre las líneas 3 y 8 corre una vez se haga click al botón.
-              </p>
-              <p>
-                Lo que hace esta función es: primero obtiene una fecha y la muestra por consola.
-                Luego calcula 10 millones de veces la fecha, y la muestra nuevamente por consola. El
-                ejemplo es claramente ridículo, pero demuestra como una operación que consume mucho
-                tiempo, inhabilita el navegador.
-              </p>
-              <p>
-                Mientras se ejecuta el loop, el control queda allí y no es devuelto al navegador
-                para continuar actualizando la pantalla o continuar ejecutando código. A esto se le
-                llama "código bloqueante".
-              </p>
+              - Un ejemplo horrible.
+              - Hablar de funciones anónimas.
+              - Este codigo bloquea al navegador
```

Comment:
```suggestion
              - Este código bloquea al navegador.
```

Y que te parece si además en esta parte preguntamos: "Y por qué creen que lo bloquea? qué opinan?" Como esperando que participen jaja